packages/web/package.json

```
@@ -75,13 +75,13 @@
     "mobx": "^6.3.10",
     "mobx-react-lite": "^3.2.2",
     "mobx-utils": "^6.0.4",
-    "next": "12.0.10",
+    "next": "13.4.9",
     "next-seo": "^6.0.0",
     "qrcode.react": "^1.0.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
```

[Next.js min react version](https://nextjs.org/docs/pages/building-your-application/upgrading/version-13) is `18.2.0`

```suggestion
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
```